### PR TITLE
add GitHub Actions to build and push Docker images

### PR DESCRIPTION
**Pull request by datelier**
SSIA

### CODE REVIEW

File: .github/workflows/update-version.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          fi
+
+          VERSION=`curl -sL "https://pypi.org/pypi/vald-client-python/json" | jq -r '.info.version'`
+          VERSION="v${VERSION}"
+          if [ "$CURRENT_VERSION" = "${VERSION}" ]; then
+            echo "not uploaded to pypi"
+            exit 0
+          fi
+
+          echo "${VERSION}" > VALD_ONNX_INGRESS_FILTER_VERSION
```
Comment (vankichi) [inline .github/workflows/update-version.yml:47]:
```suggestion
          git remote set-url origin "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
          echo "${VERSION}" > VALD_ONNX_INGRESS_FILTER_VERSION
```
the committer of the version file should be `vdaas-ci`?

File: .github/workflows/update-version.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          git checkout main
+          git config --global user.name "VDaaS org"
+          git config --global user.email "ci@vdaas.org"
+          git add VALD_ONNX_INGRESS_FILTER_VERSION
+          git commit --signoff -m ":bookmark: Release ${VERSION}"
+          git tag ${VERSION}
+          git remote set-url origin "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
```
Comment (vankichi) [inline .github/workflows/update-version.yml:54]:
```suggestion
```

File: .github/workflows/update-version.yml (hunk 1 above)
Comment (datelier) [reply .github/workflows/update-version.yml:48]: :+1:

**Review by vankichi — APPROVED** [review]

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**